app/javascript/onboarding/components/IntroSlide.jsx

```diff
@@ -50,7 +50,8 @@ class IntroSlide extends Component {
           {' '}
           <strong>quick questions</strong>
           {' '}
-          for you before you get started...
+for you before you get
```

Comment:
Heck. Looks like some changes happened on their own during a merge. I'll pop a `WIP` in the PR title until the PR is back to doing just one thing.